JDR8888/love-hungry
# Messaging

## Changes
-Login/Logout is toggled depending on user session, same goes for logout script
-checks logged_in for homepage, changes button to 'find a date' if you are logged in
-view message has a reply/and view profile button
-sushi is changed to japanese
-view profile is responsive, displays accurate info (from messaging)
-multiple messages now work, but the following command needs to be manually entered into the mysql cli:
DROP INDEX `message_sender_id_receiver_id_unique` ON `message`;
-i wrote a deploy script for heroku that includes this after seeding, hopefully this works
-timeout is set to 1min from 30sec for the db. its like 200,000 entries so takes a minute. It works on my machine, but if it persists on yours we can change the restaurant seeds from "const rawRestaurantData01 = require('./restaurantData/restaurant-list.json');" to "const rawRestaurantData01 = require('./restaurantData.json');" and that gives the smaller sample set

## How to test
-Check the nav bar after logging in and logging out
-check homepage after logging in
-check out the message tab, you might need to try logging in as different users to see them 
-run that DROP INDEX `message_sender_id_receiver_id_unique` ON `message`; in your mysql and then send multiple messages to the same person, there should be a console log showing the message sent, you can also fetch them with insomnia
-try running seeds
